test/cli/test-proj2.py

```diff
@@ -153,3 +153,13 @@ def test_gui_project_loads_absolute_vs_solution_2():
     ret, stdout, stderr = cppcheck(['--project=test.cppcheck'])
     assert ret == 0, stdout
     assert stderr == ERR_A + ERR_B
+
+def test_vs_project_missing_file():
+    with open('proj2/proj2.vcxproj', 'rt') as fin:
+        s = fin.read()
+        with open('proj2/test.vcxproj', 'wt') as fout:
```

Comment:
We should use `tmpdir` if we are generating new files but none of the tests here currently do that.